PasswordVerifier.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace PasswordVerifier
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter your password: ");
+            string password;
+            password = Console.ReadLine();
+            bool isTrue = true;
+            try
+            {
+                if (string.IsNullOrWhiteSpace(password))
+                {
+                    isTrue = false;
+                    throw new Exception("Password should not be empty");
```

Comment:
throwing exceptions at each step will not get all the errors in passwords at 1 go. 
let's say the user tries the password "ahfj",
this will give error saying 'it does not have upper case char'
again user tried "ahsADS",
this time they will get error "it does not match min password length condition"

instead, if we store all errors & throw them at once, 
user will get all the conditions that have to be met at once.